CONTRIBUTING.md

```diff
@@ -299,6 +299,19 @@ Example:
 }
 ```
 
+Overrides specifically for CodeCatalyst can be set using the `aws.dev.codecatalystService` setting. This is a JSON object consisting of keys/values required to override API calls to CodeCatalyst: `region`, `endpoint`, `hostname`, and `gitHostname`. If this setting is present, then all keys need to be explicitly provided.
+
+Example:
+
+```json
+"aws.dev.codecatalystService": {
+    "region": "us-east-1",
```

Comment:
Originally this was `us-west-2`, I'm not sure if this only works with certain regions. @justinmk3 ?

Reply:
Will update.